prov/usnic/src/usdf_msg.c

```diff
@@ -837,6 +838,7 @@ usdf_msg_handle_recv(struct usdf_domain *udp, struct usd_completion *comp)
 			rx_ptr += copylen;
 			rxlen -= copylen;
 			iov_resid -= copylen;
+			rqe->ms_resid -= copylen;
```

Comment:
This *should* be fine, but the reason the other `rqe` members are cached into local variables is to try to convince the compiler to optimize them more effectively (more likely to live in registers, etc.).  It would make sense to use that pattern for `ms_resid` as well.